src/content/subgames.cpp

```diff
@@ -67,15 +67,19 @@ SubgameSpec findSubgame(const std::string &id)
 	std::vector<GameFindPath> find_paths;
 	while (!search_paths.at_end()) {
 		std::string path = search_paths.next(PATH_DELIM);
-		find_paths.emplace_back(path + DIR_DELIM + id, false);
-		find_paths.emplace_back(path + DIR_DELIM + id + "_game", false);
+		path = path.append(DIR_DELIM).append(id);
+		find_paths.emplace_back(path, false);
+		path = path.append("_game");
```

Comment:
you don't need to affect the reference. Append modify the path object itself

http://www.cplusplus.com/reference/string/string/append/

Reply:
At least SOMETHING behaves normally in C++. As a matter of fact, I disliked the "path = path.blabla" part of the new lines. Good to see that I can get rid of them.

Reply:
Ok. Got rid of the pesky "path = path. ...".